app/server/routes/v1/user.js

```diff
@@ -228,4 +228,53 @@ router.get("/:name", async (req, res) => {
   }
 });
 
+/**
+ * @swagger
+ * paths:
+ *  /api/user/{name}:
+ *    delete:
+ *      security:
+ *        - bearerAuth: []
+ *      tags:
+ *        - User
+ *      summary: Remove user by name.
+ *      parameters:
+ *        - in: path
+ *          name: name
+ *          required: true
+ *          schema:
+ *            $ref: '#/components/schemas/User/properties/name'
+ *      responses:
+ *        '404':
+ *          description: User not found.
+ *        '401':
+ *          description: Not Authorized. Must be user to delete user.
+ *        '204':
+ *          description: Successfully removed user.
+ *        '400':
+ *          description: Bad Request.
+ */
+
+// Remove user by name
+router.delete("/:name", async (req, res) => {
+  try {
+    const user = await User.findOne({ name: req.params.name }).exec();
+
+    if (!user) return res.status(404).send("User not found.");
+
+    if (user.name !== req.user.name)
```

Comment:
Are we also going to have admins be able to 'delete' users?

Reply:
Yes eventually. The main purpose of this addition, at this stage in the project, was so that our Testers can “clean up” after their tests. If they create a test user they want to be able to remove it after. As it stands right now you have to be logged in as the user you are trying to remove.

Reply:
It’s also worth noting that in the future, documents will not be deleted. They will be “removed”, so that they seem to be deleted but really they have just been archived or hidden.